test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,76 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.tag.Tag;
+
+public class UtilsTest {
+    private static final Object NULL_OBJECT = null;
+
+    @Test
+    public void isAnyNull() {
+        ArrayList<String> emptyList = new ArrayList<String>();
```

Comment:
If we are doing the "isAnyNull() looks inside containers" thing, then this variable (`emptyList`) is not needed, since it does not tell us anything:

* If `isAnyNull()` looks inside containers, then `isAnyNull(emptyList)` will return false.
* If `isAnyNull()` does not look inside containers, then `isAnyNull(emptyList)` will still return false.